app/views/coronavirus_local_restrictions/_tier_one_restrictions.html.erb

```diff
@@ -46,7 +46,7 @@
 
   <%= render partial: "coronavirus_local_restrictions/future_restrictions", locals: { restriction: restriction } %>
 
-  <% unless restriction.present? && restriction.future_alert_level == 4 %>
+  <% if restriction.present? && restriction.future_alert_level != 4 && before_christmas? %>
```

Comment:
I found myself a bit confused by the inverting of this conditional wondering about the `restriction.present?` side of things. Is that needed anymore since the Tier 1 change? I'd guess we expect every request to this view to have a present restriction.

Reply:
You're right, the `restriction.present?` shouldn't be needed anymore, and the view ill fail on `area_name` if the restriction isn't present. I'll take it out.